finch/processes/utils.py

```diff
@@ -657,8 +657,27 @@ def dataset_to_dataframe(ds: xr.Dataset) -> pd.DataFrame:
         ds["time"] = [y.replace(hour=12) for y in time_values]
         ds.time.attrs = attrs
     df = ds.to_dataframe().reset_index()
-    new_cols = [ll for ll in ["lat", "lon", "time"] if ll in df.columns]
-    df = df.sort_values(new_cols).set_index(new_cols)
+
+    if "realization" not in ds.dims:
+        new_cols = [ll for ll in ["lat", "lon", "time"] if ll in df.columns]
+    else:
+        new_cols = [
+            ll
+            for ll in ["lat", "lon", "time", "realization", "scenario", "region"]
```

Comment:
Can't we just remove `realization` from this list instead of removing it below twice ?

Reply:
done